packages/common/src/schemas/sign-on/pickHandleSchema.ts

```diff
@@ -18,21 +16,27 @@ export const errorMessages = {
   missingHandleError: 'Please enter a handle.'
 }
 
-export const generateHandleSchema = ({
+/**
+ * Restricted handles set includes route paths which are defined at the client level and passed in
```

Comment:
Can we just define (/move) the restricted handles in common? They should be same across the two platforms right?

Reply:
I started to do that originally. They're the same but the hairy bit is that we also restrict names of the routes that we use on web, so to get it all in common we would have to move the web `routes.ts` which feels odd. @dylanjeffers debated and decided the best compromise is just to leave routes as-is and just import routes from web into mobile